### DISCUSSION TITLE
Any examples please?

### DISCUSSION THREAD

**Discussion by jbrepogmailcom** [LUCIT-Systems-and-Development/unicorn-binance-trailing-stop-loss#23] [Q&A]
Hello, I could not find any examples how to configure the bot for particular situations, May I please ask for it?

Can the bot handle these typical trailing order situations?

Situation: SELL
pair: BTCEUR
current price: 16000 EUR
I want to sell at limit price: 16100 EUR
Trailing margin: 0.1% means that with every price move up the threshold is set to new price * (1-margin). If the price drops then threshold stays the same. If the price moves below the threshold then order ie executed and BTC is sold for EUR.
If current price moves above limit price, then order is executed and BTC is sold for EUR.

Situation: BUY
pair: BTCEUR
current price: 16000 EUR
I want to buy at limit price: 15900 EUR
Trailing margin: 0.1% means that with every price move down the threshold is set to new price * (1+margin). If the price rises then threshold stays the same. If the price moves above the threshold then order ie executed and BTC is bought for EUR.
If current price moves below limit price, then order is executed and BTC is bought for EUR.

Thanks, Jan

**Comment by oliver-zehentleitner** [comment]
Hello!

Does this help you?
https://www.lucit.tech/ubtsl-cli.html
